lms/djangoapps/courseware/migrations/0002_add_coursepreference.py

```diff
@@ -1,28 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
I think we want to keep this, but we'll remove the table via another database migration.